tools/file_packager.py

```diff
@@ -1126,9 +1152,15 @@ def generate_js(data_target, data_files, metadata):
   else:
     ret += '''
     }
-    loadPackage(%s);\n''' % json.dumps(metadata)
+    await loadPackage(%s);\n''' % json.dumps(metadata)
```

Comment:
If this is the very end of the function then you can just do `return loadPackage` I think?

(doing `return` instead of `await` might also mean you don't need to make the outer function `async`).